<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Accident in USA  Predictor</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>731</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_loading">
     <attribute name="title">
      <string>Loading</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_loading">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Loading Data</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_output">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>591</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_eda">
     <attribute name="title">
      <string>EDA</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_EDA">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>201</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
     <widget class="MplWidget" name="MplWidget" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>611</width>
        <height>361</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_eda">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Severity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Temperature</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_eda_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Severity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Waether Condtion</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Weather Condtion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Road Condtion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Severity</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_eda_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Severity</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_features">
     <attribute name="title">
      <string>Feature Selection</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_feature">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>460</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Random Forest</string>
      </property>
     </widget>
     <widget class="MplWidget" name="MplWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>611</width>
        <height>361</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>611</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_modeling">
     <attribute name="title">
      <string>RF - Bagging</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>440</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Model</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Classification Report</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample Weight</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>341</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Upsampled</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Downsampled</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>380</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All features</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Top 10 features</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>360</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Feature Selection</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Confustion Matrix</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>331</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Adaptive Boosting Trees</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Confustion Matrix</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>341</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Feature Selection</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All features</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Top 10 features</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>450</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Model</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Classification Report</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>331</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
